core/build.gradle.kts

```diff
@@ -24,10 +24,7 @@ android {
 }
 
 dependencies {
-    implementation(libs.androidx.appcompat)
     implementation(platform(libs.androidx.compose.bom))
-    implementation(libs.androidx.compose.material3)
-    implementation(libs.androidx.compose.ui)
-    implementation(libs.androidx.core.ktx)
-    implementation(libs.material)
+    api(libs.androidx.compose.material3)
```

Comment:
I am not sure we should use `api` with Compose dependencies since an app without Compose can theoretically use OUDS Android.

Reply:
To be discussed.

Reply:
I agree.